docs/spec.md

```diff
@@ -84,7 +84,7 @@ completely numeric to simplify generation of StableHLO programs.
 ```ebnf
 Type         ::= ValueType | NonValueType
 ValueType    ::= TensorType | QuantizedTensorType | TokenType | TupleType
-NonValueType ::= ElementType | FunctionType | StringType
+NonValueType ::= ElementType | FunctionType | StringType | QuantizedElementType
```

Comment:
How about "ElementType | QuantizedElementType | ...", to be similar to "TensorType | QuantizedTensorType | ..." above?

Reply:
done